docs/development/getting-started.rst

```diff
@@ -13,6 +13,16 @@ dependencies, install ``cryptography`` in ``editable`` mode. For example:
     $ pip install --requirement dev-requirements.txt
     $ pip install --editable .
 
+On OS X:
+
+If you have installed OpenSSL via Homebrew you may need to set ``CFLAGS`` and  ``LDFLAGS`` environment variables before installing the ``dev-requirements.txt`` otherwise pip may fail with include errors. For example:
```

Comment:
This should probably just be rephrased to say that you *must* install OpenSSL from Homebrew or MacPorts and set those vars (or for a static build you'll need `CRYPTOGRAPHY_OSX_NO_LINK_FLAGS="1" + the absolute paths to the libraries in `LDFLAGS`).